### DISCUSSION TITLE
Licence Key Doesn't Work

### DISCUSSION THREAD

**Discussion by RBHannam** [rsayfutdinov/Pikka#50] [General]
Hi,

I bought a licence key for Pikka but it doesn't work. It says it has already been used in an activation. This is true, in the sense that I used it on my old Mac, which I have now migrated to a new one. Apart from buying a new key, is there any way to get my existing key to work? 

Thanks in advance for any and all help.

**Comment by rsayfutdinov** [comment]
Hello,

I created a promo code for you: YPPNPM79M3RL

You can download the app from mac AppStore and redeem the code there.

Regards,
Ruslan.

**Comment by RBHannam** [comment]
Thanks Russian,

That is very helpful.

Best wishes,

RBH
